Makefile

```diff
@@ -1,4 +1,4 @@
-prefix=/usr/local
+prefix=/usr
```

Comment:
again, this is unneeded.

Reply:
Oops

Reply:
Uh. What are you looking at the branch I’m merging says /usr/local

Reply:
This review is outdated and was already fixed